download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import hail as hl
+
+from hail_scripts.reference_data.constants import GCS_PREFIX
+from hail_scripts.utils.clinvar import (
+    download_and_import_latest_clinvar_vcf,
+    CLINVAR_GOLD_STARS_LOOKUP,
+)
+from hail_scripts.utils.hail_utils import write_ht
+
+CLINVAR_HT_PATH = os.path.join(GCS_PREFIX, 'clinvar/clinvar.GRCh{genome_version}.{timestamp}.ht')
+
+def run(environment: str):
+    for genome_version in ['37', '38']:
+        mt = download_and_import_latest_clinvar_vcf(genome_version)
+        timestamp = hl.eval(mt.version)
+        ht = mt.rows()
+        ht = ht.annotate(
+            gold_stars=CLINVAR_GOLD_STARS_LOOKUP.get(hl.delimit(ht.info.CLNREVSTAT))
+        )
+        ht.describe()
+        ht = ht.transmute(info=ht.info.select('ALLELEID', 'CLNSIG')).select('info', 'gold_stars')
```

Comment:
In the current implementation of the new backend, we expect the fields `clinical_significance_id`,`alleleId`, and `goldStars`, so this is close but not quite the same. `clinical_significance_id` is essentially the enum representation of CLNSIG, mapping all the strings to the much more compact integers. If we do not do this here we will have to do it during the data loading step so it seems better to me to do that mapping here and save extra storage space/ make the clinvar annotation in the pipeline as minimal as possible.

Reply:
👍 I hadn't ported over the mapping cause we hadn't figured out enum sharing yet but a dictionary is fine for now!

Reply:
Ha.  Shower thought me remembered about the delimiter.  Ignore 2!

Reply:
I think I understand what you're doing with the snake-case `_id`.  It's reserved for string mapping?  That makes sense to me.

Reply:
I do think standardizing would be nice though - there are going to be a lot of `_id` fields in our final data. Maybe we do `finalVarianbleName_id` as our naming convention, so in this case `clinicalSignificance_id`? Or if that seems way too ugly I'm open to other suggestions